specs/trust-store-trust-policy.md

```diff
@@ -53,31 +53,166 @@ $XDG_CONFIG_HOME/notation/trust-store
 The Trust store currently supports three kinds of identities, additional identities may be supported in future :
 
 - **Certificates**: The `x509/ca` trust store contains named stores that contain Certificate Authority (CA) root certificates.
-- **SigningAuthority Certificate**: The `x509/signingAuthority` trust store contains named stores that contain Siging Authority's root certificates.
+- **SigningAuthority Certificate**: The `x509/signingAuthority` trust store contains named stores that contain Signing Authority's root certificates.
 - **Timestamping Certificates**: The `x509/tsa` trust store contains named stores with Time Stamping Authority (TSA) root certificates.
 
-Any additional sub directories under names stores and certificates in it are ignored. **NOTE**: Implementation SHOULD warn if it finds sub directories with certificates under a named store, to help diagnose misconfigured store.
+Any additional sub directories under a named store and certificates in it are ignored. **NOTE**: Implementation SHOULD warn if it finds sub directories with certificates under a named store, to help diagnose misconfigured store.
 
 ## Trust Policy
 
-Users who consume signed artifact from a registry use the trust policy to specify trusted identities which will sign the artifacts, and level of signature verification to use.
+Users who consume signed artifacts from OCI registries, or signed arbitrary blobs stored on file system, use trust policies to specify trusted identities that signed the artifacts, and level of signature verification to enforce. The trust policy is a JSON document.
+
+**NOTE**: Support for verifying signed arbitrary blob has been added in policy version 1.1. While the policy version 1.1 can be used for verifying both signed OCI artifacts as well as signed arbitrary blobs, policy version 1.0 can only be used for verifying signed OCI artifacts.
 
 ### Trust Policy Schema
 
-The trust policy is a JSON document, here are some examples.
+#### Version 1.1
+
+- **`version`**(*string*): This REQUIRED property is the version of the trust policy. This MUST be either `1.1` or `1.0`.
+- **`trustPolicies`**(*string-array of objects map*): This REQUIRED property represents a collection of trust policies.
+  - **`name`**(*string*): This REQUIRED property represents the name of the trust policy.
+  - **`scopes`**(*array of strings*): This REQUIRED property acts as a policy selector and determines which trust policy is applicable for verifying a given artifact. Values MUST be prefixed with either `oci:` or `blob:` to indicate what artifact type the scope must be applied to.
```

Comment:
Can scopes value contain random prefix like `abc:sometext`? Is this a valid trust policy statement ?